sklearn/tests/test_docstring_parameters.py

```diff
@@ -199,6 +199,7 @@ def _construct_sparse_coder(Estimator):
 
 
 @pytest.mark.parametrize('name, Estimator', all_estimators())
+@ignore_warnings(category=FutureWarning)
```

Comment:
The FutureWarnings of the estimators are handled separately in the code below. I think we should do the same here.